packages/fiori/src/ShellBar.hbs

```diff
@@ -107,6 +109,7 @@
 					icon="sap-icon://search"
 					data-ui5-text="Search"
 					data-ui5-notifications-count="{{notificationsCount}}"
+					data-ui5-stable="search-button"
```

Comment:
**[suggestion (non-blocking)](https://conventionalcomments.org/#labels):** If we want to distinguish the button that executes the search, with the field that is used for the search input, then how about having this `data-ui5-stable` attribute be `"execute-search"` instead of `"search-button"` or `"search"`, so that it is similar to the `"cancel-search"` button, in that the "button" role isn't actually used in the name?